### PR TITLE
Eliminated rule info

### PR DESCRIPTION
**Pull request by dpaks**
Tested it end to end.

@ashutosh-mishra  @YogeshRajmane

### CODE REVIEW

File: gbpservice/nfp/configurator/lib/schema.py [hunk 1]
```diff
@@ -81,24 +81,10 @@
               'stitching_cidr': "",
               'stitching_interface_index': "",
               'provider_mac': "",
-              'stitching_mac': "",
```
Comment (njagadish) [inline gbpservice/nfp/configurator/lib/schema.py:84]: These changes are fine. Wonder if any of the unit test cases needs to be modified. Can we check that.

Comment (dpaks) [inline gbpservice/nfp/configurator/lib/schema.py:84]: Yes..Unit tests need to be changed. That shall be done as part of the review comments addressal branch which shall incorporate these changes. Is that fine?

Comment (njagadish) [inline gbpservice/nfp/configurator/lib/schema.py:84]: Yes. Fine. But, both of this PR and the review comment addressed PR should get merged together

**Comment by njagadish** [comment]
Changes are fine. Unit tests will break. Unit tests will be fixed in next PR.

**Pull request merged**